src/commands/application/instances/mod.rs

```diff
@@ -31,13 +31,7 @@ pub enum InstancesSubcommand {
         version: ApplicationVersion,
     },
     /// Start the interactive session to connect to the remote Elixir instance.
-    Connect {
-        /// The instance name to connect to.
-        name: String,
-        /// (optional) The port for livebook instance.
-        #[arg(long, short, default_value_t = 8080)]
-        port: u16,
-    },
+    Connect,
```

Comment:
@onimsha Do we want to provide some optional flags similar to `wukong deployment execute --namespace pro --version green`?

Reply:
@jk-gan I think we should, since it will aligns with what we're supporting in the interactive sessions of other commands as well.

Reply:
So basically, user can skip the selection by passing as a optional parameter for all including pod name?

Reply:
no, the pod name is not a parameter, only the `--namspace` and `--version`. In the event that any of these parameters are set, the CLI will skip a specific step, or both of the steps of selecting `namespace` and `version`.

Only the last step, which is selecting the pod, is mandatory in the interactive session.

Reply:
what about we still provide the current method to connect? 
For example, 
- `wukong application instances connect green@staging/mv-xxx-yyy` -> we will use the existing flow
- `wukong application instances connect --namespace staging --version green` -> we show the interactive flow, but skip the step when the info is provided from the flags
- `wukong application instances connect` -> we show the full interactive flow

Reply:
Wouldn't it be confusing to the user?

Reply:
ya, actually quite confusing haha